src/Cropper.tsx

```diff
@@ -768,9 +768,28 @@ class Cropper extends React.Component<CropperProps, State> {
       newCrop = restrictPosition(newCrop, this.mediaSize, this.state.cropSize, zoom, rotation)
     }
 
+    if (!event.repeat) {
```

Comment:
note: MDN did [mention](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/repeat#browser_compatibility) that Chrome and Opera have an [open bug](https://crbug.com/40940886) with around the `repeat` property.

If I'm understanding it correctly, it could mean that users on Windows and Linux could see multiple `onInteractionStart` calls when holding a key. I'm not sure if that is a big deal or not so I wanted to call it out.